storagegroup/storagegroup.go

```diff
@@ -68,7 +113,7 @@ func (sg *StorageGroup) SetValidationDataSize(epoch uint64) {
 func (sg StorageGroup) ValidationDataHash() (v checksum.Checksum, isSet bool) {
 	v2 := (storagegroup.StorageGroup)(sg)
 	if checksumV2 := v2.GetValidationHash(); checksumV2 != nil {
-		v.ReadFromV2(*checksumV2)
+		v.ReadFromV2(*checksumV2) // FIXME(@cthulhu-rider): #226 handle error
```

Comment:
does it relate #226? won't it be forgotten?

Reply:
Yeah Im gonna cover it in 226.